.github/PULL_REQUEST_TEMPLATE/bug-fix.md

```diff
@@ -69,7 +69,7 @@ This PR fixes #nnnn.
     other environments, add a new line with relevant details.
 -->
 
-- [ ] Paper 1.16.3 (any OS, any Java 8+ version)
+- [ ] Paper 1.16.4 (any OS, any Java 8+ version)
```

Comment:
Maybe we should consider changing this to "latest version" or something so that this doesn't need constant updating